### PR TITLE
Chore: move package files to root

### LINKED ISSUE #21: chore: move package files into root rather than /src directory
I don't remember why it's like that but it has some odd side-effects. Like setting up lint to run in the whole repo is akward since package.json isn't in the root (and configuring prettier is a mess when using the dest path as `../`) and the prepack/postpack section in package.json is a bizarre complication to get the readme into the package.

do it and merge to `beta` branch (which deploys a new `serverless-aws-static-file-handler@beta` package to npm and can test new package in examples that way.

minor.

### PR DESCRIPTION
**Pull request by piaverous**
## What did you implement:

Closes #21

## How did you implement it:
Just moved the files, and then updated some path references

## How can we verify 
* Run `npm run test` and see all is green
* Edit a file, like a small lintable change (add space, add a semi-colon...), and then run `npm run lint` and `npm run lint-fix`

## Todos:

~- [ ] Write tests~
- [X] Fix linting errors
- [x] All tests pass on our CI Server: https://travis-ci.org/activescott/serverless-aws-static-file-handler

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,17 +46,15 @@
...
+    "src/error.html",
+    "src/plugins/BinaryMediaTypes.js",
+    "src/StaticFileHandler.js"
   ],
   "scripts": {
-    "test": "nyc --cwd ./ ./node_modules/.bin/mocha",
-    "lint": "prettier -l \"../**/*.{js,md,yml,json,html}\"",
-    "lint-fix": "prettier --write \"../**/*.{js,md,yml,yaml,json,html}\"",
-    "coverage": "npm run test && nyc report --reporter=text-lcov | coveralls",
-    "prepack": "cp ../README.md ./ ; cp ../LICENSE ./",
```
Comment (piaverous) [inline package.json]: I don't think this is needed anymore, so I removed it :wave: 
When running `npm pack`  the README.md and LICENSE are in the `tgz` file. 

Leaving this in just duplicated the README & LICENSE files, that were twice in the `tgz`, both at root dir and src/

**Comment by activescott** [comment]
@piaverous Thank you for the PR! I have added a couple minor changes to get this working correctly in travis that I tried out in the beta branch to support the semantic-release deploys. As soon as the checks pass I'll merge. Thanks again!

File: package.json (hunk 1 above)
Comment (activescott) [reply package.json]: That's correct. It is not needed. This was a wonky hack to support the files being in `./src/`

**Pull request merged**